pkg/controller/bootstrap/jws.go

```diff
@@ -34,17 +34,17 @@ func computeDetachedSig(content, tokenID, tokenSecret string) (string, error) {
 
 	signer, err := jose.NewSigner(jose.HS256, jwk)
 	if err != nil {
-		return "", nil
+		return "", fmt.Errorf("can't make a HS256 signer from the given token")
```

Comment:
SGTM to append the source error there as well. I just didn't understand why no errors were there in the first place so added some `fmt.Errorf`s.